src/app/subject/store/sensors-data/sensors-data.effects.ts

```diff
@@ -97,7 +97,7 @@ export class SensorsDataEffects {
       !timeFramesEqual(timeFrame, prevTimeFrame) ||
       timeInterval !== prevTimeInterval
         ? this.store.dispatch(new actions.Load())
-        : []
+        : this.store.dispatch(new actions.TimeIntervalNoChange())
```

Comment:
Looks good to me. I just have one question. I could see the definition of `nochange` actions and them being fired here. How exactly we handle/receive these events?

Reply:
This just leaves a 'trail' in the store and does nothing. So after the SetTimeInterval action -> Load action can be dispatched or TimeIntervalNoChange action can be dispatched. If time interval or time frame is unchanged and a Load action is not dispatched, I feel that having SetTimeInterval action as the last action dispatched (and nothing after) is somewhat incomplete. Although, a handling can be implemented like letting the user know about it. What do you think?

Reply:
Ok. Thanks for clarifying. I just wanted to understand the reason behind it. I don't think we should let the user know that there is no change, since it is obvious. So if there is nothing  that needs to be done in particular we can leave it as it is.

Reply:
Ok thanks